tests/test_dark.py

```diff
@@ -0,0 +1,224 @@
+        error
+        """
+        mean = afwMath.makeStatistics(self.exposure.getImage(), afwMath.MEAN).getValue()
+        sigma = afwMath.makeStatistics(self.exposure.getImage(), afwMath.STDEV).getValue()
+        print("5.2", mean, sigma)
+        self.assertLess(np.abs(mean), sigma)
+
+    def test_independentFrameSigma(self):
+        """Amp sigma against readnoise.
+
+        Notes
+        -----
+        DMTN-101 5.3:
+
+        Confirm that the 5-sigma clipped standard deviation of each
+        amplifier is within 5% of the nominal readnoise, as
+        determined by a robust measure of the noise in the serial
+        overscan
+
+        """
+        ccd = self.exposure.getDetector()
+        for amp in ccd:
+            ampExposure = self.exposure.Factory(self.exposure, amp.getBBox())
+            statControl = afwMath.StatisticsControl(5.0, 5)
+            statControl.setAndMask(self.exposure.mask.getPlaneBitMask(["SAT", "BAD", "NO_DATA"]))
+            sigma = afwMath.makeStatistics(ampExposure.getImage(),
+                                           afwMath.STDEVCLIP, statControl).getValue()
+            # needs to be < 0.05
+            print("5.3", amp.getName(), sigma, amp.getReadNoise(),
+                  np.abs(sigma - amp.getReadNoise())/amp.getReadNoise())
+            self.assertLess(np.abs(sigma - amp.getReadNoise())/amp.getReadNoise(), 0.71)
+
+    def test_amplifierSigma(self):
+        """Clipped sigma against CR-rejected sigma.
+
+        Notes
+        -----
+        DMTN-101 5.4:
+
+        Run a CR rejection on the result and confirm that the
+        unclipped standard deviation is consistent with the 5-sigma
+        clipped value.
+
+        """
+        crTask = RepairTask()
+        crRejected = self.exposure.clone()
+        psf = measAlg.SingleGaussianPsf(21, 21, 3.0)
+        crRejected.setPsf(psf)
+        crTask.run(crRejected, keepCRs=False)
+
+        ccd = self.exposure.getDetector()
+        for amp in ccd:
+            ampExposure = self.exposure.Factory(self.exposure, amp.getBBox())
+            clipControl = afwMath.StatisticsControl(5.0, 5)
+            clipControl.setAndMask(self.exposure.mask.getPlaneBitMask(["SAT", "BAD", "NO_DATA"]))
+            sigmaClip = afwMath.makeStatistics(ampExposure.getImage(),
+                                               afwMath.STDEVCLIP, clipControl).getValue()
+
+            crAmp = crRejected.Factory(crRejected, amp.getBBox())
+            statControl = afwMath.StatisticsControl()
+            statControl.setAndMask(self.exposure.mask.getPlaneBitMask(["SAT", "BAD", "NO_DATA", "CR"]))
+            sigma = afwMath.makeStatistics(crAmp.getImage(), afwMath.STDEV, statControl).getValue()
+            # needs to be < 0.05
+            print("5.4", amp.getName(), sigma, sigmaClip, np.abs(sigma - sigmaClip)/sigmaClip)
+            self.assertLess(np.abs(sigma - sigmaClip)/sigmaClip, 3.0)
+
+    def test_55(self):
```

Comment:
Why 150? The beginning of Sec. 5 of DMTN-1010 mentions 100 dark exposures. For the LATISS data that you are using, I'm guessing that there are not enough darks to do this test, correct? If so, ow will it be implemented in the future? With different data?

Reply:
(same comment for test_56 and test_57)

Reply:
All of the notes are taken directly from the DMTN-101 text.  This one is on page 4 of the 2018-11-26 draft.  In any case, I'm removing tests that aren't implementable at the moment, so this will go away.